patches/chrome-test-base-in_process_browser_test.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/test/base/in_process_browser_test.cc b/chrome/test/base/in_process_browser_test.cc
+index 13aebd52d7e2734156cbe68b1d39211ee7a48677..a0ae3fe29bb97204cb0777c4d1c102cfced60c23 100644
+--- a/chrome/test/base/in_process_browser_test.cc
++++ b/chrome/test/base/in_process_browser_test.cc
+@@ -348,6 +348,7 @@ void InProcessBrowserTest::Initialize() {
+ InProcessBrowserTest::~InProcessBrowserTest() = default;
+ 
+ void InProcessBrowserTest::SetUp() {
++  IN_PROCESS_BROWSER_TEST_SET_UP
```

Comment:
`#define SetUpCommandLine(COMMAND_LINE) SetUpCommandLine(COMMAND_LINE); BraveBrowser::SuppressBrowserWindowClosingDialogForTesting(true)`

Reply:
Fixed in 3169e02f2a674c55732f7dd6cac4f8a8082d2025.